src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -133,8 +257,50 @@ public Command setWristPositionCommand(DoubleSupplier posDeg) {
      * @return A {@link RunCommand} to change the wrist setpoint by velocityDegPerSec.
      * Resets the wrist dampening profile after completion.
      */
-    public Command changeWristGoalPosition(double velocityDegPerSec) {
-        return setWristPositionCommand(() -> wristInputs.wristGoal + (velocityDegPerSec * UPDATE_PERIOD))
-            .finallyDo(wrist::resetProfile);
+    public Command wristChangeGoalPosition(double velocityDegPerSec) {
+        return wristSetPositionCommand(() -> wristInputs.wristGoal + (velocityDegPerSec * UPDATE_PERIOD))
+            .finallyDo(wrist::wristResetProfile);
+    }
+
+    public Command runElevatorCharacterization() {
+    return Commands.sequence(
```

Comment:
All of the arm components have extension limits that we will likely run into in the course of system identification. Are we allowed to oscillate the mechanism (move forwards *and* backwards) over the course of several seconds so we can still collect the data we need but don't violate motion constraints?